test/contains-test.js

```diff
@@ -0,0 +1,110 @@
+var tape = require("tape"),
+    array = require("d3-array"),
+    d3 = require("../");
+
+tape("a sphere contains any point", function(test) {
+  test.ok(d3.geoContains({type: "Sphere"}, [0, 0]));
+  test.end();
+});
+
+tape("a point contains itself (and not some other point)", function(test) {
+  test.ok(d3.geoContains({type: "Point", coordinates: [0, 0]}, [0, 0]));
+  test.ok(d3.geoContains({type: "Point", coordinates: [1, 2]}, [1, 2]));
+  test.equal(false,d3.geoContains({type: "Point", coordinates: [0, 0]}, [0, 1]));
+  test.equal(false,d3.geoContains({type: "Point", coordinates: [1, 1]}, [1, 0]));
```

Comment:
When using test.equal (and similar methods), the intended order of arguments is *actual*, *expected*; see [tape’s README](https://github.com/substack/tape/blob/master/readme.markdown#tequalactual-expected-msg).

And for consistency I would write both positive and negative tests the same way, so:

```js
test.equal(d3.geoContains(…), true);
test.equal(d3.geoContains(…), false);
```